### DISCUSSION TITLE
[ArisStudio.0.2.22] Apostrophe Not Registered in Dialogue

### DISCUSSION THREAD

**Discussion by NoelHarkov** [Tualin14/baPlayer#29] [Q&A] [answered]
I been facing this issue for the whole day, seeing the apostrophe unusable in text in the 2.2.2 version. I know the standard is normally be `\'` like `Hifumi\'s` to generate `Hifumi's` in text. This somehow works in 0.3.0a2 (I have issue loading resources somehow, no clue what is the reason) but not in 2.2.2. I wonder if it is just an oversight and not yet addressed in 2.2.2 for EN users since its original users were all Chinese, hence the `'` can be used solely for the quotation coding.
Did I missed something in this, or is it an actual problem in 2.2.2?

**Comment by kiraio-moe** [comment, marked as answer]
Yes, it's the problem that v2.x got. v2.x didn't recognize `'` or `"` as the part of a word. It's already fixed in v0.3.x.

As for v0.3.x, they're currently in unstable state.

**Reply by NoelHarkov** [reply]
I see. Thanks for the reply. I will still avoid using v0.3.x for now because I somehow have issue loading the data. Not sure what mistake I did in setting up the root for it. Is there any specific to installing the data?
If the latest build is unstable, is there any plan to setup a hotfix for v2.x for now?

**Comment by kiraio-moe** [comment]
> I see. Thanks for the reply. I will still avoid using v0.3.x for now because I somehow have issue loading the data.

Can you give me more info on this, like a screenshot of the log screen (it's at the top of Aris Studio, hover your mouse over it)?
So I can better understand what the problem is going on and hopefully can help you.

> Is there any specific to installing the data?

Not any, it's just that the file structure in v0.3.x is different from v2.2.x. Here's the current Aris Studio resources file structure:
```
{root}
|_ audio
   |_ bgm
      |_ Theme_01.ogg
      |_ Theme_02.ogg
      |_ ...
   |_ sfx
      |_ SE_49mm_01.wav
      |_ SE_AirRelease_01.wav
      |_ ...
|_ character
   |_ spr
      |_ airi_spr.atlas
      |_ airi_spr.png
      |_ airi_spr.skel
|_ image
   |_ background
      |_ BG_AbandonedCorridor.jpg
      |_ BG_Amusement.jpg
      |_ ...
   |_ foreground
      |_ Event01_Airi.PNG
      |_ Event01_Akane.PNG
      |_ ...
   |_ midground (Don't know what this is used for)
```

> If the latest build is unstable, is there any plan to setup a hotfix for v2.x for now?

Not anytime soon, because I don't have time to do it. Sorry 😞

**Reply by NoelHarkov** [reply]
![Aris Studio_2023-07-08-01-34-37_1280x720](https://github.com/Tualin14/ArisStudio/assets/114988570/5e97d1d7-6bfa-4f52-b060-bebfaa9d5fbb)
![Aris Studio_2023-07-08-01-34-12_1280x720](https://github.com/Tualin14/ArisStudio/assets/114988570/0d49b08e-1e12-4915-b01c-33f66534bd93)
Thanks for the reply. These are the logs that I take note of. I tried running the Hifumi demo and yeah, the resources are not loading somehow. I didn't do any modification other than removing the banner, so it's pretty weird. It worked in the previous v0.2.x version, so the load isn't an issue. The data loading format is right considering I just copied the resources from v0.2.x and followed the exact folder arrangement.
>Not anytime soon, because I don't have time to do it. Sorry 😞

Don't worry, I understand if you and Tualin have other commitments. Your contribution to this project is more than I expected already. Literally didn't know this existed as I did fanfiction scenes through manual video editing.

**Comment by ttttttttttttttllllllllllllll** [comment]
I've temporarily released a changed version of 0.2 that you can use.
You can see the changes on the release page. [0.2.23](https://github.com/Tualin14/ArisStudio/releases/tag/ArisStudio.0.2.23)

Sorry I still have a very large amount of work to deal with right now. orz

**Reply by NoelHarkov** [reply]
Thank you! It's okay, take care out there. A temp fix is still better than nothing. Good luck in your work!
